framework/resources/Datasources/ctas/DRILL-1248/group_by.ddl

```diff
@@ -0,0 +1,4 @@
+DROP TABLE IF EXISTS dfs.drillTestDir.`DRILL-1248/group_by_parquet`;
+create table dfs.drillTestDir.`DRILL-1248/group_by_parquet`
```

Comment:
A minor request for change - can we have the SQL statements in one case format? It's all caps above for DROP TABLE and all lower case for CREATE TABLE